reftests/src/helpers.rs

```diff
@@ -125,12 +127,32 @@ impl coset::TaggedCborSerializable for Payload {
 }
 
 /// Returns the key pair, KID and the header for a key seed.
-pub fn generate_key(key_seed: u8) -> (ed25519_dalek::Keypair, Vec<u8>, Header) {
-    let mut seed = [0u8; 32];
-    seed[31] = key_seed;
-
-    let mut prng = rand::rngs::StdRng::from_seed(seed);
-    let ed25519_key = ed25519_dalek::Keypair::generate(&mut prng);
+pub fn generate_key(
+    key_seed: Option<u8>,
+    pem: Option<String>,
+) -> (ed25519_dalek::Keypair, Vec<u8>, Header) {
+    let ed25519_key = if let Some(pem) = pem {
```

Comment:
Make 2 functions with this, one for pem and one for key seed, and then use both in `generate_key`.

Reply:
👍